flattener_bench_test.go

```diff
@@ -0,0 +1,156 @@
+package quamina
```

Comment:
Add the benchmarks I am presenting, I think it will be good to commit those (if you agree I'll move `status.json` to `testdata` directory)

What do you think?

Reply:
+1 yes please.

Reply:
do we want to run the benchmarks in CI? I am not sure about it, since I don't github actions will give us good results and will take some time to run the benchmarks.

Reply:
See issue #30 - one problem is that GitHub runs actions single-threaded on a pathetic tiny instance. So we probably can't routinely run them all.  This bothers me a bit because I want to make sure that nothing gets checked in that accidentally causes a noticeable slowdown.